frontend/src/views/teacher/discussions/ShowClarifications.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="clarification-container" v-if="clarifications.length > 0">
+    <div class="discussion">
+      <ul>
+        <li
+          style="margin-bottom: 25px !important;"
+          v-for="clarification in clarifications"
+          :key="clarification.id"
+        >
+          <div class="text-left">
+            <b
+              >{{ clarification.userName }} replied on
+              {{ clarification.date }} :</b
+            >
+            <span v-html="convertMarkDown(clarification.message)" />
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div v-else>
+    <v-card-text class="text-center">
+      No clarifications yet
+    </v-card-text>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit, Watch } from 'vue-property-decorator';
+import Discussion from '@/models/management/Discussion';
+import User from '@/models/user/User';
+import { convertMarkDown } from '@/services/ConvertMarkdownService';
+import Reply from '@/models/management/Reply';
+@Component
+export default class ClarificationComponent extends Vue {
+  @Prop() readonly clarifications!: Reply[];
+  user: User = this.$store.getters.getUser;
```

Comment:
not used, and similar to a vue in student, why not have a discussions package that have all components and reduce repetition